rapids-cmake/cpm/thrust.cmake

```diff
@@ -98,7 +98,7 @@ function(rapids_cpm_thrust NAMESPACE namespaces_name)
   endif()
 
   # only install thrust when we have an in-source version
-  if(Thrust_SOURCE_DIR AND _RAPIDS_INSTALL_EXPORT_SET)
+  if(Thrust_SOURCE_DIR AND _RAPIDS_INSTALL_EXPORT_SET AND NOT exclude)
```

Comment:
Just making sure, this is equivalent to the `to_exclude` var that's defined in other CMake files now, right? We just don't define that variable here because it's not needed elsewhere?

Reply:
Correct